README.rst

```diff
@@ -199,12 +200,12 @@ Developers". Feel free to cite our work in your research using the following Bib
                       Quinn, Jamie and
                       Wessel, Paul},
       title        = {{PyGMT: A Python interface for the Generic Mapping Tools}},
-      month        = dec,
-      year         = 2022,
+      month        = march,
+      year         = 2023,
       publisher    = {Zenodo},
-      version      = {0.8.0},
-      doi          = {10.5281/zenodo.7481934},
-      url          = {https://doi.org/10.5281/zenodo.7481934}
+      version      = {0.9.0},
+      doi          = {10.5281/zenodo.7772533},
+      url          = {https://doi.org/10.5281/zenodo.7772533}
     }
 
 To cite a specific version of PyGMT, go to our Zenodo page at
```

Comment:
A little unrelated, but could you change L215 to mention `dimfilter` (wrapped in #1492) has a dedicated citation too?

```
Note that some modules like ``dimfilter``, ``surface`` and ``x2sys`` also have their dedicated citation.
```